cmd/zc_traverser_local.go

```diff
@@ -369,6 +427,194 @@ func (t *localTraverser) Traverse(preprocessor objectMorpher, processor objectPr
 		return fmt.Errorf("failed to scan path %s due to %s", t.fullPath, err.Error())
 	}
 
+	t.hashTargetChannel = make(chan string, 1_000) // "reasonable" backlog
```

Comment:
The logic here and below is quite extensive, is there a reason we didn't abstract this out as a helper method? it would make much more readable.